.github/workflows/CI.yml

```diff
@@ -0,0 +1,224 @@
+name: "IGListKit CI"
```

Comment:
Remove this since you created a separate PR here.

Reply:
> I just found out that all the files are deleted and re-added, this lost all the previous git history or blame, could you use the "git mv" or similar to mark the file as a move instead of "delete+add"? The goal is to keep the git history and blame for each file so that we know who was the original author. Thanks!

Sorry for the mess, it was a hard time put those together and I moved files around with different combinations. I think I found a better solution for it, please see #1487. I tried to keep changes as small as possible.

Reply:
PS. About CI: while there is another open #1478 only for CI support, this PR was based on it since I need some way to verify my changes. If I remove support (in #1487 I didn't add it at all, for now)  how we will see the progress/status? Or your plan is to land #1478 first?